sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java

```diff
@@ -195,11 +197,12 @@ public <T> Coder<T> coderFor(
    *       the lexicographically smallest {@link Class#getName() class name} being used.
    * </ul>
    */
-  public static CoderRegistry createDefault() {
-    return new CoderRegistry();
+  public static CoderRegistry createDefault(@Nullable SchemaRegistry schemaRegistry) {
+    return new CoderRegistry(schemaRegistry);
   }
```

Comment:
This might be a breaking change for users

Can we have the old createDefault() method as well and have it return new CoderRegistry(null)?

Would maintain existing use cases and limit the number of files in this PR